package/MDAnalysis/coordinates/TRR.py

```diff
@@ -24,6 +24,10 @@ class TRRWriter(XDRBaseWriter):
     used by other Gromacs tools to store and process other data such as modes
     from a principal component analysis.
 
+    If the data dictionary of a TimeStep contains the key 'lambda' the
+    corresponding value will be used as the lambda value for written TRR file.
+    If None is found the lambda is set to 1.
```

Comment:
In Gromacs, `lambda == 0` typically means "full interaction" so a default of 0 would be better.

This is also the default in Gromacs TRRs:
```
md.trr frame 0:
   natoms=    140246  step=         0  time=0.0000000e+00  lambda=         0
```